packages/lesswrong/components/messaging/InboxNavigation2.tsx

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import { Components, registerComponent } from "../../lib/vulcan-lib";
+import { UseMultiResult } from "../../lib/crud/withMulti";
+
+const styles = (theme: ThemeType): JssStyles => ({
+  noConversationsMessage: {
+    padding: 16,
+  }
+})
+
+// The Navigation for the Inbox components
+const InboxNavigation2 = ({
+  conversationsResult,
+  currentUser,
+  selectedConversationId,
+  setSelectedConversationId,
+  classes,
+}: {
+  conversationsResult: UseMultiResult<"ConversationsList">;
+  currentUser: UsersCurrent;
+  title?: JSX.Element | String;
+  selectedConversationId: string | undefined;
+  setSelectedConversationId: React.Dispatch<React.SetStateAction<string | undefined>>;
+  classes: ClassesType;
+}) => {
+  const { results: conversations, loading, loadMoreProps } = conversationsResult;
+
+  const {
+    ConversationItem2,
+    Loading,
+    SectionFooter,
+    Typography,
+    LoadMore,
+  } = Components;
+
+  return <>
+      {conversations?.length ? (
+        conversations.map((conversation, idx) => (
+          <ConversationItem2
+            key={conversation._id + idx}
+            conversation={conversation}
+            currentUser={currentUser}
+            selectedConversationId={selectedConversationId}
+            setSelectedConversationId={setSelectedConversationId}
+          />
+        ))
+      ) : loading ? (
+        <Loading />
+      ) : (
+        <Typography variant="body2" className={classes.noConversationsMessage}>
+          You are all done! You have no more open conversations.
```

Comment:
I could be wrong, but I seem to remember @agnesstenlund being generally anti-exclamation marks?

Reply:
yes 😍 this instance I like it a bit better than other ones, but still prefer without. Thanks @oetherington!

Reply:
I also remember seeing this before ever having any conversations, could that be true? Wondering if the most common time to see this is _before_ having conversations (in which case we should have a "start a new conversation" CTA in the empty state), vs. when you've archived all your conversations? which doesn't feel like a thing, i.e. this is not like an email where you try to reach inbox zero?

Reply:
> Wondering if the most common time to see this is before having conversations
👍 This will now be basically the only time you'll see this, as I removed the ability to archive conversations (we decided this on the retreat if you remember, because it was very rarely used and old conversations go to the bottom of the pile anyway)

I've changed it to "You have no open conversations." instead (perhaps a but terse)